garden-backend-service/src/models/user.py

```diff
@@ -0,0 +1,12 @@
+from uuid import UUID
+
+from sqlalchemy.orm import Mapped, mapped_column
+
+from .base import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    username: Mapped[str]
+    identity_id: Mapped[UUID] = mapped_column(unique=True)
```

Comment:
Is this all data that we derive from the Globus Auth token? If so, what do we gain from persisting this?

Reply:
Exactly yeah this is just what we get at minumum from the globus auth token, and my sense is that it's likely to be handy if/when we need to integrate with other globus stuff. Didn't read too deeply into it past the first sentence here  https://docs.globus.org/api/auth/specification/#using_identities

Reply:
no immediate plans to do anything but store it Just In Case